### DISCUSSION TITLE
Add new custom page to Azuracast

### DISCUSSION THREAD

**Discussion by freqmand** [AzuraCast/AzuraCast#5098] [Q&A]
is it possible to add a new custom page html or php to Azuracast installation ? example https://myStation.com/myCustomPage . 
is there an example ?
thanks.

**Comment by xewl** [comment]
Have a look at https://docs.azuracast.com/en/developers/plugins

**Reply by freqmand** [reply]
Thanks for your reply, 
The page I want to add is not related to Azura, let's assume it's a simple < p >Hello World</ p > page. 
I think it should have a custom nginx configuration and adding new volume that map to the new html file. But I'm not sure how to do this.
looks like the plugin concept is a bit complex for me and I'm not sure if this is what I'm looking for..

**Reply by xewl** [reply]
Azura isn't meant or built to be sharing other pages/applications.
Nginx + docker config might be a solution but I personally wouldn't go that route.
I run my Azura instances on subdomains. (:

**Comment by freqmand** [comment]
any update please ? 
@SlvrEagle23 
@Vaalyn
